src/app/integrations/create-page/flow-view/flow-view.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+import { Integration } from '../../../store/integration/integration.model';
+
+@Component({
+  selector: 'ipaas-integrations-flow-view',
+  templateUrl: './flow-view.component.html',
+  styleUrls: ['./flow-view.component.scss']
+})
+export class FlowViewComponent implements OnInit {
+
+  @Input() integration: Observable<Integration>;
+  i: Integration;
+
+  ngOnInit() {
+    this.integration.subscribe((integration) => {
```

Comment:
Ah yeah, I think I hadn't yet arrived at the `getOrCreate()` function in the stores, so this was my way of getting around the fact that we're either creating an empty integration object or working with one that's been fetched.  Also I haven't 100% decided how much this controller is going to handle vs the containing page, so just left this code in place for now.